### DISCUSSION TITLE
Does transloco supports multiple scope definitions?

### DISCUSSION THREAD

**Discussion by kumart-netapp** [ngneat/transloco#596] [Q&A]
Does transloco supports multiple scope definitions?

**Comment by ES-Six** [comment]
Yes, we use this feature.

In our angular module we declare scopes like this at the provider level :
```ts
import { TRANSLOCO_SCOPE } from '@ngneat/transloco';

...

providers: [
    ...
    { provide: TRANSLOCO_SCOPE, useValue: ['scope1', 'scope2', 'scope3'] },
    ...
  ],
```

Then we use scopes like this :
```html
<ng-container *transloco="let t">
 <div>{{ t('scope1.something') }}</div>
 <div>{{ t('scope2.something') }}</div>
</ng-container>
```

Note 1 : by default, trandloco will prefix translation keys by their scope.
Note 2 : tested successfully with lazy loaded modules and also with standard modules and with multi module project.
